### PR TITLE
More minigames & Abandoned Crate Improvements PT 1 (Reflection Golf)

### PR DESCRIPTION
**Pull request by AtnerNT**
This PR is the first of a few but this ended up having alot of changes so I am splitting it into several PRs so its easier to review.

This first PR adds a new minigame, reflection golf that will be the first of a few new minigames added to abandoned crates as well as new sprites for abandoned crates.

Following PRs will add more minigames to abandoned crates

### Changelog:

CL: [New] Added 'Reflection Golf' Minigame to abandoned crates.
CL: [Improvement] Added new spritework for abandoned crates
CL: [Fix] Fixed abandoned crates not unlocking on game completion
CL: [Fix] Fixed abandoned crates not being able to be hand interacted with to be opened/closed
CL: [Balance] Mapped a small amount of abandoned crates onto Relicta, Valley, Ancient Station, Ancient Station & Maintrooms, replacing some loot that was previously just in regular chests or on floor.

### CODE REVIEW

**Comment by Bod9001** [comment]
Looks good

**Comment by AtnerNT** [comment]
Ready for review/merge

File: CodeScanList.json [hunk 1]
```diff
@@ -551,7 +551,9 @@
             },
 			"Path" : {
 				"Methods": [
-					"string Combine(string, string)"
+					"string Combine(string, string)",
+                    "string Combine(string, string, string)",
+                    "string Combine(string, string, string, string)"
```
Comment (MaxIsJoe) [inline CodeScanList.json:556]: Codescan changes must be done in a separate PR

File: UnityProject/Assets/Scripts/MiniGames/MiniGameModules/ReflectionGolf/ReflectionGolfInput.cs [hunk 2]
```diff
@@ -0,0 +1,261 @@
+using MiniGames.MiniGameModules;
+using UnityEngine;
+using System;
+using UI.Minigames;
+
+public class ReflectionGolfInput
```
Comment (MaxIsJoe) [inline UnityProject/Assets/Scripts/MiniGames/MiniGameModules/ReflectionGolf/ReflectionGolfInput.cs]: namespace

File: UnityProject/Assets/Scripts/MiniGames/MiniGameModules/ReflectionGolf/ReflectionGolfLevel.cs [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+		public void LoadLevelFromFile(string levelName)
+		{
+			miniGameModule.ClearAllMoves();
+
+			string path = Path.Combine("MiniGamesData", "ReflectionGolf", $"{levelName}.txt");
+
+			if (AccessFile.Exists(path) == false)
+			{
+				Loggy.LogError($"MiniGames/RelfectionGolfLevel.cs at line 61. The specified file path does not exist! {path}");
```
Comment (MaxIsJoe) [inline UnityProject/Assets/Scripts/MiniGames/MiniGameModules/ReflectionGolf/ReflectionGolfLevel.cs]: follow our standard for writing logs
https://unitystation.github.io/unitystation/contribution-guides/Development-Standards-Guide/#0-in-code-documentation-and-clarity

**Review by MaxIsJoe — CHANGES_REQUESTED** [review]

**Comment by AtnerNT** [comment]
If there's no further reviews, this is ready to merge now.

**Review by MaxIsJoe — APPROVED** [review]

**Pull request merged**